docs/common/agent-builder/_behavior-trees.md

```diff
@@ -0,0 +1,42 @@
+A behavior tree is a decision-making model used to determine which actions an agent (such as a character in a video game) should take in different situations.
+You can think of a behavior tree as a flow chart that the AI follows, choosing appropriate actions based on what it knows about itself and its environment.
+A behavior tree is composed of nodes organized into a hierarchical structure which starts with a root node and branches out to form different paths.
+Each path represents a line of decision-making that results in one or more actions for the AI to perform.
+
+Each node in the tree executes logic and returns one of the following statuses as a result:
+
+- **Running** -> The task represented by this node's branch hasn't yet completed.
+- **Success** -> The node has completed its task successfully.
+- **Failure** -> The node has failed its task. This doesn't necessarily mean an error occurred. For example, a heal action might fail if the character doesn't have any potions.
+
+Different types of nodes have different responsibilities and may react differently to the statuses of other nodes.
+
+- **Root Node**: The beginning of the tree, where the AI starts its decision-making.
+  Its job is simply to execute whatever node is below it in the tree and return its status.
+- **Sequence Node**: This node has one or more children and acts as a logical AND.
+  It executes its child nodes sequentially, and if any one of them fail then the Sequence Node skips execution of the remaining children and returns a failure status.
```

Comment:
```suggestion
  It executes its child nodes sequentially, and if any one of them fail then the Sequence Node immediately returns the Failure status and stops executing child nodes.
```

Reply:
Updated, kept the wording similar to the updated verbiage for Selector Node